instances.go

```diff
@@ -348,11 +430,11 @@ func (c *Client) SetInstancePolicies(ctx context.Context, policies MultiplePolic
 
 	if policies.AllowedNetwork != nil {
 		policy := InstancePolicy{
-			PolicyType: AllowedNetwork,
+			PolicyType: "allowedNetwork",
```

Comment:
This got reset again
```suggestion
			PolicyType: AllowedNetwork,
```